examples/README.md

```diff
@@ -12,14 +12,19 @@ Currently, the following scenarios are included:
 
 ### Lodging
 
-#### Shopping
+- [
+  `AvailabilityCalendarsQuickStartScenario.java`](src/main/java/com/expediagroup/sdk/xap/examples/scenarios/lodging/AvailabilityCalendarsQuickStartScenario.java):
```

Comment:
Let's move example of AvailabilityCalendar API behind Quotes

Reply:
It's in alphabetic order and aligns with the order in the file system. Do we need to customize the order here?

Reply:
done